bundles/org.openhab.binding.e3dc/src/main/resources/ESH-INF/thing/e3dc-strings-channel-types.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="e3dc"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+	<channel-type id="string1-dc-voltage-channel">
+		<item-type>Number:ElectricPotential</item-type>
+		<label>String 1 Potential</label>
+		<description>Volt on String 1</description>
```

Comment:
See Readme and below.
```suggestion
		<description>Voltage on String 1</description>
```

Reply:
Fixed in Readme and xmls - Voltage, Current and Power